runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -3610,12 +3610,10 @@ typedef enum {
   am_opShutdown,                           // signal main process for shutdown
 } amOp_t;
 
-#ifdef CHPL_COMM_DEBUG
 static inline
 chpl_bool op_uses_on_bundle(amOp_t op) {
   return op == am_opExecOn || op == am_opExecOnLrg;
 }
-#endif
```

Comment:
Why is this now required for non-debug builds too? (I see that call that uses it, but I'm wondering why the logic changes if this was mostly a non-functional code cleanup)

Reply:
Ah, never mind -- I think it's just that we've lost context that was available at the call site inside the callee, so we need to use this to query now.

Reply:
Right, the new call to this is in the common code where the operator could be anything, and as it happens is the first call to it outside of debug context.